src/Elrond/Serialization.cpp

```diff
@@ -46,13 +49,25 @@ sorted_json preparePayload(const Elrond::Proto::TransactionMessage& message) {
         {"gasLimit", json(message.gas_limit())},
     };
 
+    if (!message.sender_username().empty()) {
+        payload["senderUsername"] = json(TW::Base64::encode(TW::data(message.sender_username())));
```

Comment:
```suggestion
        payload["senderUsername"] = json(Base64::encode(TW::data(message.sender_username())));
```
`TW::` usually can be omitted

Reply:
Indeed, fixed.